recipes/xtr/config.yml

```diff
@@ -3,9 +3,5 @@ versions:
     folder: all
   "2.0.1":
     folder: all
-  "2.0.0":
-    folder: all
   "1.0.1":
     folder: all
-  "1.0.0":
-    folder: all
```

Comment:
What's the reason for deleting versions? I think it would be better to leave them in, to avoid breaking anyone's build.

Reply:
This does not really directly affect the availability of these versions, they simply become "unmaintained". Updates to the recipe are published as new revisions, all old revisions are kept intact.

The standard practice on CCI is to keep all major versions, if possible, and a few latest minor versions. Only the latest patch version is typically kept around. This is done to limit the size of the CI builds, mostly, and to reduce maintenance burden if the recipes require a lot of patching.

https://github.com/conan-io/conan-center-index/blob/master/docs/adding_packages/sources_and_patches.md#supported-versions